client/cpp/kserve-api/README.md

```diff
@@ -525,7 +525,7 @@ Usage:
 - Usage Example
 
 ```Bash
-./grpc_async_infer_resnet --images_list resnet_input_images.txt --labels_list resnet_labels.txt --grpc_port 9111
+./grpc_async_infer_resnet --images_list ../../../common/resnet_input_images.txt --labels_list ../../../common/resnet_labels.txt --grpc_port 9000
```

Comment:
did u verify if output is correct? the output below is:
- airliner
- bee
- gorilla
- ...

but the .txt input file is:
- airliner
- arctic-fox
- bee
- ...

best option would be to manually rerun and validate